src/dimindexing.jl

```diff
@@ -1,6 +1,16 @@
+"""
+    dim_to_indices(arr::IndexedDimsArray{T, N}, inds::Tuple{Vararg{Any, N}}) where {T, N}
+
+Convert indexes into the IndexedDimsArray into indexes into the underlying array.
+
+```jldoctest; setup = :(using IndexedDims)
+julia> arr = IndexedDimsArray([4.0, 5.0], ["a", "b"]);
+
+julia> IndexedDims.dim_to_indices(arr, (["b"],))
+([2],)
+```
+"""
 function dim_to_indices(arr::IndexedDimsArray{T, N}, inds::Tuple{Vararg{Any, N}}) where {T, N}
```

Comment:
This seems like a weird name for this function?

Reply:
Yep

Reply:
`parent_indices`?

Reply:
https://github.com/invenia/IndexedDims.jl/issues/18